draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -490,17 +490,16 @@ RESET\_STREAM frame.
 QUIC allows an application to abort reading the stream and specify an error code
 {{Section 3.5 of !RFC9000}}. An RTP receiver can signal this to the RTP sender
 by sending a QUIC STOP\_SENDING frame, which is exported to the RTP sender
-through the ROQ API, as described in {{api-considerations}}. The RTP-sender
-SHOULD not interpret the reception of STOP\_SENDING as an indication that the
-RTP receiver lost interest in the media stream as a whole, but rather that a
-part of the media stream was not received timely. In this case, the late data
-could also delay the following media frames due to head-of-line blocking, which
-the sender can avoid by transmitting new media frames on new QUIC streams. A
-sender that receives STOP\_SENDING MUST NOT retransmit any media frames already
-sent on the QUIC stream, on which it received the STOP\_SENDING, on a new QUIC
-stream. An RTP sender SHOULD instead continue to send media frames on new QUIC
-streams starting with the first frame that was not transmitted on the stream
-that received STOP\_SENDING.
+through the ROQ API, as described in {{api-considerations}}. STOP\_SENDING is
+not an indication that the RTP receiver lost interest in the media stream as a
+whole, but rather that a part of the media stream was not received timely. In
```

Comment:
OK, this is super awkward. I think we've gotten confused about the difference between what a **QUIC receiver** thinks it means when it sends "STOP\_SENDING", and what an **RTP receiver** thinks it means. 

From [RFC 9000](https://datatracker.ietf.org/doc/html/rfc9000#name-solicited-state-transitions): 

> If the stream is in the "Recv" or "Size Known" state, the transport SHOULD signal this by sending a STOP_SENDING frame to prompt closure of the stream in the opposite direction. This typically indicates that the receiving application is no longer reading data it receives from the stream, but it is not a guarantee that incoming data will be ignored.

So, it looks like the application HAS lost interest - because the media frames are arriving late enough that they are not useful. The application is sending "STOP\_SENDING" to trigger an automatic "RESET\_STREAM".

What's confusing, is that the **RTP receiver** - the application - hasn't lost interest in the **media** - it's lost interest in **getting outdated and unusable media frames.** The RTP receiver wants the RTP sender to start over, with the most current media frames, on a stream that doesn't already hold additional outdated frames. 

Does that make sense?

Assuming so, the text in this part of the document should reflect that distinction.

Reply:
Yes, that sounds correct.